<ui version="4.0" >
 <class>SearchViewBase</class>
 <widget class="QWidget" name="SearchViewBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Search</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="searchFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item row="6" column="0" colspan="3" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>1</number>
        </property>
        <property name="spacing" >
         <number>3</number>
        </property>
        <item row="0" column="1" >
         <widget class="QDateEdit" name="dateAfterButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="dateBeforeBox" >
          <property name="whatsThis" >
           <string>Include only mails before this date.</string>
          </property>
          <property name="text" >
           <string comment="Before 20th of July" >Before</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QDateEdit" name="dateBeforeButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QCheckBox" name="dateAfterBox" >
          <property name="whatsThis" >
           <string>Include only mails after this date.</string>
          </property>
          <property name="text" >
           <string comment="e.g. After 20th of July" >After</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1" colspan="2" >
       <widget class="QLineEdit" name="bodyLine" >
        <property name="whatsThis" >
         <string>Input text that must occur in the body.  Bear in mind that searching the body can be very slow.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QComboBox" name="mailbox" >
        <property name="focusPolicy" >
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="whatsThis" >
         <string>Select the mailbox to search for mail in.</string>
        </property>
        <item>
         <property name="text" >
          <string>Inbox</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Outbox</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Drafts</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Sent</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Trash</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="TextLabel1_2_2_2" >
        <property name="text" >
         <string>To</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="TextLabel1_2" >
        <property name="text" >
         <string>Flag</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QLineEdit" name="fromLine" >
        <property name="whatsThis" >
         <string>Input text that must occur in the from field.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="TextLabel1" >
        <property name="text" >
         <string>In</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="nameLabel" >
        <property name="text" >
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="TextLabel1_2_2_3" >
        <property name="text" >
         <string>Subj</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2" >
       <widget class="QLineEdit" name="subjectLine" >
        <property name="whatsThis" >
         <string>Input text that must occur in the subject.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="TextLabel1_2_2_4" >
        <property name="text" >
         <string>Body</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="TextLabel1_2_2" >
        <property name="text" >
         <string>From</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QComboBox" name="status" >
        <property name="focusPolicy" >
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="whatsThis" >
         <string>Find mail with a matching flag.</string>
        </property>
        <item>
         <property name="text" >
          <string>Any</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Read</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Unread</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Replied</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1" colspan="2" >
       <widget class="QLineEdit" name="toLine" >
        <property name="whatsThis" >
         <string>Input text that must occur in the to field.</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2" >
       <widget class="QLineEdit" name="nameLine" >
        <property name="whatsThis" >
         <string>Set the name for your query.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mailbox</tabstop>
  <tabstop>status</tabstop>
  <tabstop>fromLine</tabstop>
  <tabstop>toLine</tabstop>
  <tabstop>subjectLine</tabstop>
  <tabstop>bodyLine</tabstop>
  <tabstop>dateAfterBox</tabstop>
  <tabstop>dateBeforeBox</tabstop>
  <tabstop>nameLine</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dateAfterBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateAfterButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>28</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel" >
     <x>110</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateBeforeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateBeforeButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>42</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel" >
     <x>104</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
